dmri_seg.yml

```diff
@@ -0,0 +1,14 @@
+name: dmri_seg
+channels:
+  - conda-forge
```

Comment:
https://github.com/ANTsX/ANTs?tab=readme-ov-file#advanced-normalization-tools

I confirm that conda-forge now supplies ants.